src/masternodes/mn_checks.cpp

```diff
@@ -1069,7 +1069,7 @@ class CCustomTxApplyVisitor : public CCustomTxVisitor {
 
         if (tokenId && token.IsDAT() && IsEVMEnabled(height, mnview, consensus)) {
```

Comment:
Need to do a check for result of CreateToken to ensure that the token to be created passes validation on the DVM side first.

Reply:
Note that this is currently on master.

Reply:
tokenId here is checking the result of CreateToken.